library/include/G4CMPEmpiricalLindhardNIEL.hh

```diff
@@ -0,0 +1,75 @@
+/***********************************************************************\
+ * This software is licensed under the terms of the GNU General Public *
+ * License version 3 or later. See G4CMP/LICENSE for the full license. *
+\***********************************************************************/
+
+/// \file library/include/G4CMPEmpiricalLindhardNIEL.hh
+/// \brief NIEL calculation from Ionization yield measurement 
+/// in a germanium CDMSlite detector using photo-neutron sources.
+///
+/// Computation of NIEL according to Lindhard (Lewin Smith) yield with 
+/// a parameter k that depends on the energy.
```

Comment:
This description is incomplete, and I'm not sure it's accurate.  You've said in the DMC meeting that this model allows the user to specify their own K value, without regard for any material or theory calculation, or it allows the user to request a hard-coded simple function K(E), again without regard to the material.  If that's correct, that's what the description should say.